react/src/app/ThemeSelect/themes.js

```diff
@@ -0,0 +1,177 @@
+export const light = {
```

Comment:
These should be generated dynamically from `@momentum-ui/tokens` => `semanticColor.json`. See `@cantina/state/theme/utils.js`

Reply:
I've discussed with @Coread that we'll do this after the new token scheme goes in, rather than adapting it to the old one and then the new one later.